src/lib.rs

```diff
@@ -54,11 +64,20 @@ mod helpers;
 mod html_root_url;
 mod markdown_deps;
 
+// Ensure that at least one feature is enabled
+#[cfg(not(any(feature = "contains_regex", feature = "html_root_url_updated",
+    feature = "markdown_deps_updated")))]
+std::compile_error!("Please select at least one feature.");
+
+#[cfg(feature = "contains_regex")]
 pub use crate::contains_regex::check_contains_regex;
+#[cfg(feature = "html_root_url_updated")]
 pub use crate::html_root_url::check_html_root_url;
+#[cfg(feature = "markdown_deps_updated")]
 pub use crate::markdown_deps::check_markdown_deps;
 
 /// Assert that dependencies on the current package are up to date.
+/// Requires the "markdown_deps_updated" feature.
```

Comment:
This line shows up in the overview page:

![Screenshot from 2021-09-14 22-17-33](https://user-images.githubusercontent.com/89623/133327835-7bc1a106-59b6-4e22-8868-029890c69c58.png)

Could you instead add it as the last paragraph in the docstring so that it only shows up on the per-function and per-macro pages?

Reply:
Should be fixed now